src/commands.js

```diff
@@ -444,6 +444,41 @@ FFZ.prototype.get_banned_users = function() {
 }
 
 
+
+ // ---------------------
+ // Group Chat Renaming
+ // ---------------------
+FFZ.chat_commands.renamegroup = function(room, args) {
+	// Get the current room.
+	var controller = utils.ember_lookup('controller:chat'),
```

Comment:
The first argument room is the room model. You can just use that rather than looking up the current room from the chat controller.